app/src/forms/admin/routes.js

```diff
@@ -65,18 +390,149 @@ routes.get('/users/:userId', async (req, res, next) => {
 //Form componets help info
 //
 
+/**
+ * @openapi
+ * /admin/formcomponents/proactivehelp/object:
+ *  post:
+ *   tags:
+ *    - Admin
+ *   summary: create or update proactive help details form.io componen
+ *   description: This endpoint will create or update proactive help details form.io component
+ *   security:
+ *    - openId: []
+ *   requestBody:
+ *    required: true
+ *    content:
+ *      application/json:
+ *        schema:
+ *          $ref: '#/components/requestBodies/AdminReqProactiveHelpObject'
+ *   responses:
+ *    '200':
+ *      description: Success
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/responses/responseBody/AdminGetProactiveHelpObject'
+ *    '403':
+ *      $ref: '#/components/responses/Error/AccessDenied'
+ *    '5XX':
+ *      $ref: '#/components/responses/Error/UnExpected'
+ *    '422':
+ *      $ref: '#/components/responses/Error/UnprocessableEntityDB'
```

Comment:
For some reason this is showing up in the docs as "422 Unauthorized". It should be "403 Access Denied"?

Reply:
403 access denied is if the token has expired, but showcase used 422.

Reply:
We definitely have some cleanup to do!